platform/landing-page/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -20,7 +20,7 @@
         <property name="path" value=""/>
     </bean>
     <service id="resources" ref="resourceMapping"
-             interface="org.ops4j.pax.web.extender.whiteboard.ResourceMapping"/>
+             interface="org.ops4j.pax.web.service.whiteboard.ResourceMapping"/>
```

Comment:
❓ There are still two references to the `org.ops4j.pax.web.extender.whiteboard` package in DDF, both for the `DefaultResourceMapping` class. Should those have been updated to this new package as well?